lenet/index.html

```diff
@@ -71,6 +71,10 @@ <h5>Prediction Result:</h5>
         </div>
       </div>
     </div>
+    <script>
+      // To fix issues of loading all 3rd-party libraries when we reuse codes for electron.
+      if ( typeof module === "object") module = undefined;
```

Comment:
ditto for coding style.

Reply:
I have updated codes as your suggestions, PTAL again, thanks! @huningxin